packages/@ember/-internals/glimmer/lib/components/checkbox.ts

```diff
@@ -33,10 +33,41 @@ import layout from '../templates/empty';
 */
 const Checkbox = EmberComponent.extend({
   layout,
+
+  /**
+    By default, this component will add the `ember-checkbox` class to the component's element.
+
+    @property classNames
+    @type Array | String
+    @default ['ember-checkbox']
+    @public
+   */
   classNames: ['ember-checkbox'],
 
   tagName: 'input',
 
+  /**
+    By default this component will forward a number of arguments to attributes on the the
+    component's element:
+
+    * type
+    * checked
+    * indeterminate
+    * disabled
+    * tabindex
+    * name
+    * autofocus
+    * required
+    * form
+
+    When invoked with `{{input type="checkbox"}}`, you can only customize these attributes. When
+    invoked with `<Input @type="checkbox" />`, you can just use HTML attributes directly.
```

Comment:
This is somewhat confusing, because it seems to suggest you can do `<Input type="checkbox" />`